source/rust_verify_test/tests/common/mod.rs

```diff
@@ -457,7 +457,10 @@ pub fn relevant_error_span(err: &Vec<DiagnosticSpan>) -> &DiagnosticSpan {
         return e;
     }
     err.iter()
-        .filter(|e| e.label != Some(vir::def::THIS_PRE_FAILED.to_string()))
+        .filter(|e| {
+            e.label != Some(vir::def::THIS_PRE_FAILED.to_string())
+                && e.label != Some("type invariant declared here".to_string())
```

Comment:
This special case should probably not belong here.
(I'm posting this as a single comment because I chanced upon it while trying to look at tests.)

Reply:
do you have a suggested alternative?

Reply:
I took another look and your suggestion made complete sense this time. Not sure what I was thinking earlier.